crates/atuin/locales/app.yml

```diff
@@ -0,0 +1,81 @@
+_version: 2
+<enter>:
+  en: <enter>
+  ga: <iontráil>
+<esc>:
+  en: <esc>
+  ga: <éalaigh>
+<tab>:
+  en: <tab>
+  ga: <táb>
+Drawing inspector, but no stats:
+  en: Drawing inspector, but no stats
+  ga: Ag tarraingt cigire, ach níl statisticí ann
+Nothing to inspect:
+  en: Nothing to inspect
+  ga: Níl aon rud le scrúdú
+SRCH:
+  en: SRCH
+  ga: LORG
+"The inspector is new - please give feedback (good, or bad) at https://forum.atuin.sh":
+  en: "The inspector is new - please give feedback (good, or bad) at https://forum.atuin.sh"
+  ga: "Tá an cigire nua - tabhair aiseolas le do thoil (go maith nó go holc) ag https://forum.atuin.sh"
+delete:
+  en: delete
+  ga: scrios
+edit:
+  en: edit
+  ga: cuir in eager
+exit:
+  en: exit
+  ga: scoir
+inspect:
+  en: inspect
+  ga: scrúdaigh
+invalid tab index on input:
+  en: cluaisín neamhbhailí ar ionchur
+  ga: invalid tab index on input
+keys.ctrl-and:
+  en: <ctrl-%{key}>
+  ga: <ctrl-%{key}>
+search:
+  en: search
+  ga: cuardach
+should have been handled!:
+  en: should have been handled!
+  ga: ba chóir a bheith láimhseáilte!
+Search:
+  en: Search
+  ga: Cuardach
+Inspect:
+  en: Inspect
+  ga: Scrúdaigh
+history count:
+  en: history count
+  ga: comhaireamh na staire
+"%{time} ago":
+  en: "%{time} ago"
+  ga: "%{time} ó shin"
+time.y:
```

Comment:
this is maybe excessive, as SI units are supposed to have the same representation in every language, but it gives an idea how this could work

Reply:
(and non-SI units like days, hours, minutes, etc. do vary)